pkg/kubelet/prober/worker.go

```diff
@@ -62,6 +63,11 @@ type worker struct {
 	resultRun int
 }
 
+const (
+	// jitter factor for worker.doProbe sync loop
+	workerProbeJitterFactor = 1.0
```

Comment:
The jitter factor could be user configurable. Not in this PR but later.

@timstclair Would that be enough to sleep for a random time before entering to for loop to finish this PR? Are there are sync loops or places worth jittering?